lib/google/longrunning/operations_client.rb

```diff
@@ -96,17 +90,27 @@ def initialize \
           scopes: ALL_SCOPES,
           client_config: {},
           timeout: DEFAULT_TIMEOUT,
-          app_name: "gax",
-          app_version: Google::Gax::VERSION
+          app_name: nil,
+          app_version: nil,
+          lib_name: nil,
+          lib_version: ""
         # These require statements are intentionally placed here to initialize
         # the gRPC module only when it's required.
         # See https://github.com/googleapis/toolkit/issues/446
         require "google/gax/grpc"
         require "google/longrunning/operations_services_pb"
 
-        google_api_client = "#{app_name}/#{app_version} " \
-          "#{CODE_GEN_NAME_VERSION} gax/#{Google::Gax::VERSION} " \
-          "ruby/#{RUBY_VERSION}".freeze
+
+        if app_name || app_version
+          warn "`app_name` and `app_version` are no longer being used in the request headers."
+        end
+
+        google_api_client = "gl-ruby/#{RUBY_VERSION}"
+        google_api_client << " #{lib_name}/#{lib_version}" if lib_name
+        google_api_client << " gapic/0.1.0 gax/#{Google::Gax::VERSION}"
```

Comment:
I assumed the GAPIC version here was the version of the code that generated this file. Is that not the case?